README.md

```diff
@@ -101,12 +108,14 @@ from scratch.
 
 For this you'll need `.service` file to define systemd service. The easiest way
 to get one is to run [`gen_systemd_service_file.py`](gen_systemd_service_file.py).
-You should specify service name as a first argument and optionally provide target protocol
-as a second argument, e.g.:
+You should specify service name as an argument. Note that there are three
+predefined services for `tezos-node`: `tezos-node-{mainnet, carthagenet, delphinet}`.
```

Comment:
What if I want to run `tezos-node` in another network? E. g. create my own or connect to one created by a friend of mine.